ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+[download]: https://bintray.com/dv8fromtheworld/maven/JDA/_latestVersion
+[guild]: https://discord.gg/0hMr4ce0tIk3pSjp
+[stack overflow]: https://stackoverflow.com/questions/tagged/java
+
+## General Troubleshooting
+
+Hey there! Before you report a bug or suggest a new feature,
+  please make sure to follow these steps first!
+  
+- [ ] I have checked for similar issues.
+- [ ] I have updated to the [latest JDA version][download].
+- [ ] I have checked the branches or the staff's PRs for upcoming features/bug fixes.
+
+> This is not the place to learn Java. Please refer to [StackOverflow][stack overflow]
+  for your general programming questions.
+
+## Bug Report
+
+Provide your current **build number** and a **brief summary** of your issue.
+
+## Feature Request
+
+Provide a **specific description** of the feature you are looking for and potential **use cases**.
```

Comment:
This should rather be:
### Type
- [ ] Bug Report
- [ ] Feature Request

### Description

Replace this sentence with a description and include as many details as you think are needed to understand your issue.